nicobar_islands.html

```diff
@@ -74,6 +74,25 @@
         </div>
 
     </div>
+  
+    <div>  
+      <br/><br/>
+        <strong>The Tribes of Andaman </strong> <br/>
+        <p>The Andaman Islands are home to four 'Negrito' tribes – the Great Andamanese, Onge, Jarawa and Sentinelese. The Nicobar Islands are home to two 'Mongoloid' tribes – the Shompen and Nicobarese. </p>
+
+        <p>The 'Negrito' tribes are believed to have arrived in the islands from Africa up to 60,000 years ago. All are nomadic hunter-gatherers, hunting wild pig and monitor lizard, and catching fish with bows and arrows. They also collect honey, roots and berries from the forest. </p>
+
+        <p>The 'Mongoloid' tribes probably came to the islands from the Malay-Burma coast several thousand years ago. </p>
+
+        <p>Great Andamanese: Population 43. The tribe that has suffered most from contact with outsiders: 99% have been wiped out since the British first colonized the islands. Before the tsunami they lived in a government settlement and were dependent on government aid. They have just been moved to the islands' capital, Port Blair, as their village was badly damaged in the tsunami. </p>
+
+        <p>Onge: Population 100. Their forest home has been plundered by poachers and loggers. They were settled by the Indian administration and are dependent on food handouts. The 73 Onge living at Dugong Creek fled to high ground when they saw the sea level fall, and so survived. </p>
+
+        <p>Jarawa: Population about 270. Have only had peaceful contact with outsiders for six years. They live on the west coast of South and Middle Andaman, and are believed to have survived the tsunami. They are still completely independent and live entirely by hunting, gathering and fishing. The main threat to their existence comes from the highway running through their territory: the Indian government was ordered to close this by the Supreme Court in 2002, but it has ignored the order. </p>
+
```

Comment:
Please add information on these tribes as well Sentinelese, Shompen and Nicobarese.
Also seperate the title of each tribe with the text in the line below.

Reply:
added the new tribe info